utils/_context/containers.py

```diff
@@ -684,10 +695,11 @@ def post_start(self):
         # https://github.com/DataDog/system-tests/issues/2799
         if self.library == "nodejs":
             with open(self.healthcheck_log_file, mode="r", encoding="utf-8") as f:
-                data = json.load(f)
+                lib = json.load(f)["library"]
```

Comment:
Could you let it as `data`. in a near future we'll need other info that will be at the root of `data`.

If repeated `data["library"]` hurts, you can add `library_data=data["library"]` after.

Reply:
I was mostly trying to save horizontal space as I don't know how to break a line of code in multiple lines in Python 😅

Reply:
> I was mostly trying to save horizontal space as I don't know how to break a line of code in multiple lines in Python

Don't hurt yourself, let `./format.sh` take care of this 😄